src/libraries/System.Diagnostics.TraceSource/ref/System.Diagnostics.TraceSource.cs

```diff
@@ -219,55 +219,55 @@ public abstract partial class TraceListener : System.MarshalByRefObject, System.
         public virtual void Close() { }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
-        public virtual void Fail(string message) { }
-        public virtual void Fail(string message, string detailMessage) { }
+        public virtual void Fail(string? message) { }
+        public virtual void Fail(string? message, string? detailMessage) { }
         public virtual void Flush() { }
-        protected virtual string[] GetSupportedAttributes() { throw null; }
-        public virtual void TraceData(System.Diagnostics.TraceEventCache eventCache, string source, System.Diagnostics.TraceEventType eventType, int id, object data) { }
-        public virtual void TraceData(System.Diagnostics.TraceEventCache eventCache, string source, System.Diagnostics.TraceEventType eventType, int id, params object[] data) { }
-        public virtual void TraceEvent(System.Diagnostics.TraceEventCache eventCache, string source, System.Diagnostics.TraceEventType eventType, int id) { }
-        public virtual void TraceEvent(System.Diagnostics.TraceEventCache eventCache, string source, System.Diagnostics.TraceEventType eventType, int id, string message) { }
-        public virtual void TraceEvent(System.Diagnostics.TraceEventCache eventCache, string source, System.Diagnostics.TraceEventType eventType, int id, string format, params object[] args) { }
-        public virtual void TraceTransfer(System.Diagnostics.TraceEventCache eventCache, string source, int id, string message, System.Guid relatedActivityId) { }
-        public virtual void Write(object o) { }
-        public virtual void Write(object o, string category) { }
-        public abstract void Write(string message);
-        public virtual void Write(string message, string category) { }
+        protected virtual string[]? GetSupportedAttributes() { throw null; }
+        public virtual void TraceData(System.Diagnostics.TraceEventCache? eventCache, string source, System.Diagnostics.TraceEventType eventType, int id, object? data) { }
+        public virtual void TraceData(System.Diagnostics.TraceEventCache? eventCache, string source, System.Diagnostics.TraceEventType eventType, int id, params object[]? data) { }
```

Comment:
Ditto.

I'll stop commenting on these... please check all such object[]s to see if they should be able to contain nullable objects.